doc/runtime-information.rst

```diff
@@ -4,46 +4,60 @@ Run-time Information
 =====================
 
 Your app should run in a bundle exactly as it does when run from source.
-However, you may need to learn at run-time
-whether the app is running from source, or is "frozen" (bundled).
-For example, you might have
-data files that are normally found based on a module's ``__file__`` attribute.
-That will not work when the code is bundled.
+However, you may want to learn at run-time whether the app is running from
+source or whether it is bundled. You can use the following code to check "are
+we bundled?"::
```

Comment:
The additional question is okay, But please keep "frozen" and the reference to `__file__`.

Reply:
I find the reference to `__file__` counterproductive as it suggests that one has to use a if-else check to use it with PyInstaller. The headline referring to `__file__` is just below and there it is explained that in most cases, using `__file__` should just work.